adam/ontology/phase1_ontology.py

```diff
@@ -434,22 +546,270 @@ def subtype(sub: OntologyNode, _super: OntologyNode) -> None:
             bigger_than(_BIRD_SCHEMA_TORSO, _BIRD_SCHEMA_HEAD),
             bigger_than(_BIRD_SCHEMA_TORSO, _BIRD_LEGS),
             supports(_BIRD_LEGS, _BIRD_SCHEMA_TORSO),
-            supports(_BIRD_SCHEMA_TORSO, _BIRD_SCHEMA_HEAD),
-            supports(_BIRD_SCHEMA_TORSO, _BIRD_SCHEMA_TAIL),
-            supports(_BIRD_SCHEMA_TORSO, _BIRD_WINGS),
+            supports(
+                _BIRD_SCHEMA_TORSO,
+                [
+                    _BIRD_SCHEMA_HEAD,
+                    _BIRD_SCHEMA_TAIL,
+                    _BIRD_SCHEMA_LEFT_WING,
+                    _BIRD_SCHEMA_RIGHT_WING,
+                ],
+            ),
         ]
     ),
 )
 
-_BALL_SCHEMA = ObjectStructuralSchema(BALL)
-_BOX_SCHEMA = ObjectStructuralSchema(BOX)
-_WATER_SCHEMA = ObjectStructuralSchema(WATER)
-_JUICE_SCHEMA = ObjectStructuralSchema(JUICE)
-_BOX_SCHEMA = ObjectStructuralSchema(BOX)
-_MILK_SCHEMA = ObjectStructuralSchema(MILK)
-_DOOR_SCHEMA = ObjectStructuralSchema(DOOR)
-_HAT_SCHEMA = ObjectStructuralSchema(HAT)
-_COOKIE_SCHEMA = ObjectStructuralSchema(COOKIE)
+# schemata describing the hierarchical physical structure of objects
+_HOUSE_SCHEMA_LEFT_WINDOW = SubObject(_WINDOW_SCHEMA)
+_HOUSE_SCHEMA_RIGHT_WINDOW = SubObject(_WINDOW_SCHEMA)
+_HOUSE_SCHEMA_DOOR = SubObject(_DOOR_SCHEMA)
+_HOUSE_SCHEMA_ROOF = SubObject(_ROOF_SCHEMA)
+_HOUSE_SCHEMA_WALL = SubObject(_WALL_SCHEMA)
+
+# House modeled after a simple 1 story home as commonly seen in child's books
+# Stick example below -- ASCII art perhaps isn't the best demonstration form
+#      / \
+#    /     \
+#  /         \
+# /           \
+# -------------
+# | []  _  [] |
+# [----| |----]
+_HOUSE_SCHEMA = ObjectStructuralSchema(
+    HOUSE,
+    sub_objects=[
+        _HOUSE_SCHEMA_DOOR,
+        _HOUSE_SCHEMA_LEFT_WINDOW,
+        _HOUSE_SCHEMA_RIGHT_WINDOW,
+        _HOUSE_SCHEMA_ROOF,
+        _HOUSE_SCHEMA_WALL,
+    ],
+    sub_object_relations=sub_object_relations(
+        [
+            contacts(
+                [
+                    _HOUSE_SCHEMA_DOOR,
+                    _HOUSE_SCHEMA_ROOF,
+                    _HOUSE_SCHEMA_RIGHT_WINDOW,
+                    _HOUSE_SCHEMA_LEFT_WINDOW,
+                ],
+                _HOUSE_SCHEMA_WALL,
+            ),
+            supports(_HOUSE_SCHEMA_WALL, _HOUSE_SCHEMA_ROOF),
+            above(
+                _HOUSE_SCHEMA_ROOF,
+                [
+                    _HOUSE_SCHEMA_WALL,
+                    _HOUSE_SCHEMA_LEFT_WINDOW,
+                    _HOUSE_SCHEMA_RIGHT_WINDOW,
+                    _HOUSE_SCHEMA_DOOR,
+                ],
+            ),
+            bigger_than(
+                [_HOUSE_SCHEMA_WALL, _HOUSE_SCHEMA_ROOF],
+                [
+                    _HOUSE_SCHEMA_DOOR,
+                    _HOUSE_SCHEMA_RIGHT_WINDOW,
+                    _HOUSE_SCHEMA_LEFT_WINDOW,
+                ],
+            ),
+        ]
+    ),
+)
+
+# schemata describing the hierarchical physical structure of objects
```

Comment:
I think this comment ended up in many places due to cut-and-paste. We should replace it with something more specific, e.g. 
```
# sub-object structure of a car
```